src/thorlabs_cube/driver/base.py

```diff
@@ -10,6 +10,9 @@
 
 
 class _Cube:
+    _CHANNEL: int = 0x01
+    _RESERVED: int = 0x00
```

Comment:
```suggestion
    _RESERVED: int = 0x00
    _CHANNEL: int = 0x01
```
Ascending numeric order.